packages/dropdowns/examples/field-variants.md

```diff
@@ -0,0 +1,211 @@
+        <FormField>
+          <Toggle
+            checked={state.isCompact}
+            onChange={e => setState({ isCompact: e.target.checked })}
+          >
+            <FormLabel>Compact</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.isBare} onChange={e => setState({ isBare: e.target.checked })}>
+            <FormLabel>Bare</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.focusInset}
+            onChange={e => setState({ focusInset: e.target.checked })}
+          >
+            <FormLabel>Focus inset</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.disabled} onChange={e => setState({ disabled: e.target.checked })}>
+            <FormLabel>Disabled</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={state.placeholder}
+            onChange={e => setState({ placeholder: e.target.checked })}
+          >
+            <FormLabel>Placeholder</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <FormField>
+          <Toggle checked={state.message} onChange={e => setState({ message: e.target.checked })}>
+            <FormLabel>Message</FormLabel>
+          </Toggle>
+        </FormField>
+        <StyledSpacer />
+        <Dropdown selectedItem={state.validation} onSelect={validation => setState({ validation })}>
+          <Field>
+            <Label>Validation</Label>
+            <Select isCompact>{state.validation}</Select>
+          </Field>
+          <Menu isCompact>
+            <Item value="none">none</Item>
+            <Item value="success">success</Item>
+            <Item value="warning">warning</Item>
+            <Item value="error">error</Item>
+          </Menu>
+        </Dropdown>
+        <StyledSpacer />
+        <FormField>
+          <Toggle
+            checked={!!state.inline}
+            onChange={event => setState({ inline: event.target.checked })}
+          >
+            <FormLabel>Inline</FormLabel>
+            <FormHint>See example code for inline styling overrides</FormHint>
+          </Toggle>
+        </FormField>
+      </Well>
+    </Col>
+    <Col size={8}>
+      <Dropdown>
+        <StyledField>
+          <StyledLabel isRegular={state.regularWeightLabel}>Select dropdown</StyledLabel>
+          {state.showHint && <StyledHint>Hint</StyledHint>}
+          <Select
+            isCompact={state.isCompact}
+            isBare={state.isBare}
+            focusInset={state.focusInset}
+            disabled={state.disabled}
+          >
+            Example
+          </Select>
+          {state.message && (
+            <StyledMessage validation={state.validation !== 'none' && state.validation}>
```

Comment:
> VM3593 index.js:1 Warning: Failed prop type: Invalid prop `validation` of value `false` supplied to `ForwardRef`, expected one of ["success","warning","error"].

It looks like a boolean is passed to the validation prop, but the value should be one of `success`, `warning`, or `error`.

Reply:
I've changed this to a ternary.